/wbc/wbc:
  ros__parameters:
    control_rate: 250.0
    robot_model_config:
        type: rbdl
    task_config:
        names: ["ee_pose"]
        ee_pose:
          type: 1
          priority: 0
          root: world
          tip: link_ee
          ref_frame: world
          weights: [1.0,1.0,1.0,1.0,1.0,1.0]
          activation: 1.0
    solver_config:
        type: qpoases
    scene_config:
        type: velocity_qp
    integrate: true
